src/Raven.Server/Documents/Indexes/Persistence/Lucene/LuceneIndexWriter.cs

```diff
@@ -106,7 +106,7 @@ private void CreateIndexWriter()
             using (indexWriter.MergeScheduler)
             {
             }
-            indexWriter.SetMergeScheduler(new ErrorLoggingConcurrentMergeScheduler());
+            indexWriter.SetMergeScheduler(new SerialMergeScheduler());
```

Comment:
ConcurrentMergeScheduler spawns threads for merging and we:
1) have only one write transaction that would be accesed concurrently by those threads
2) using ThreadLocal inside lucene voron directory

I will add a TODO to my list to check what we can do about this, but I was getting NRE here due to ThreadLocal usage